packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/submit/controller.js

```diff
@@ -0,0 +1,10 @@
+const BackOfficeV2Service = require('../../../../../../services/back-office-v2');
+
+const backOfficeV2Service = new BackOfficeV2Service();
+
+/** @type {import('express').Handler} */
+exports.post = async (req, res) => {
+	await backOfficeV2Service.submitQuestionnaire(req.params.caseReference);
+
+	res.sendStatus(200);
```

Comment:
There didn't seem to be any need to respond to the front end with the mapped data, which is what the old thing used to do. I can add that in if we'd prefer though